pkg/querier/queryrange/split_by_interval.go

```diff
@@ -169,7 +171,6 @@ func (h *splitByInterval) Do(ctx context.Context, r queryrange.Request) (queryra
 	if err != nil {
 		return nil, err
 	}
-
 	return h.merger.MergeResponse(resps...)
```

Comment:
Sorry I wanted to also tackle the fact that I've seen an empty list going there once. But then I realized I'm not sure why this has happened so I let it be a 500, because we should take care of it.